packages/keystone/kv.js

```diff
@@ -63,7 +63,12 @@ function getKVClient (name = 'default', purpose = 'regular', opts = { kvOptions:
     if (!KV_CLIENTS[clientKey]) {
         const kvEnvName = `${name.toLowerCase()}_KV_URL`
         const redisEnvName = `${name.toUpperCase()}_REDIS_URL`
-        const kvUrl = conf[kvEnvName] || conf[redisEnvName] || conf.KV_URL || conf.REDIS_URL
+        let kvUrl = conf[kvEnvName] || conf[redisEnvName] || conf.KV_URL || conf.REDIS_URL
+        try {
+            kvUrl = JSON.parse(kvUrl)
+        } catch (err) {
+            // skip that error
```

Comment:
Add more detailed description, its a bit hard to understand whats going on without context of PR